build.sbt

```
@@ -57,19 +57,20 @@ def projectWithPlayVersion(majorMinorVersion: String) =
     libraryDependencies ++= Seq(
       "com.gu.play-secret-rotation" %% "core" % "0.17",
       "org.typelevel" %% "cats-core" % "2.0.0",
+      "com.typesafe.akka" %% "akka-http-core" % "10.2.0",
```

For some reason we get compilation errors when we don't make this dependency explicit:

`/code/play-googleauth/play-v28/src/test/scala/com/gu/googleauth/GoogleAuthTest.scala:6:13: object http is not a member of package akka`

---

Nice spot! Ideally, I think if this was _just_ needed to make the tests compile, it would have made sense to restrict this to being a [test-scoped dependency](https://www.scala-sbt.org/1.x/docs/Library-Dependencies.html#Per-configuration+dependencies), ie:

```suggestion
      "com.typesafe.akka" %% "akka-http-core" % "10.2.0" % Test,
```